index.js

```diff
@@ -1,8 +1,10 @@
 const ChangeTypes = require('./lib/ChangeTypes');
+const factor = require('./lib/factor');
 const simplifyExpression = require('./lib/simplifyExpression');
 const solveEquation = require('./lib/solveEquation');
 
 module.exports = {
+  factor,
```

Comment:
So the reasoning I had was that you can't factor an equation, you can solve for x. So I would expect us to factor within solveEquation when solving an equation that can only be solved by factoring (a quadratic equation)